depot.json

```diff
@@ -0,0 +1 @@
+{"id":"x19jffd9zf"}
```

Comment:
This is the ID of your Depot project in the PostHog org - Kyle has sent @timgl an invite to the org, and he can invite anyone else who needs access 👍